playbooks/provisioning/openstack/sample-inventory/group_vars/OSEv3.yml

```diff
@@ -10,6 +10,9 @@ openshift_master_cluster_public_hostname: "{{ groups.lb.0|default(groups.masters
 
 osm_default_node_selector: 'region=primary'
 
+openshift_hosted_router_wait: True
+aopenshift_hosted_registry_wait: True
```

Comment:
I think this is a typo - "openshift_hosted_registry_wait"?

Reply:
Ah indeed!